internal/backends/winit/event_loop.rs

```diff
@@ -478,21 +471,14 @@ impl EventLoopState {
             }),
 
             Event::AboutToWait => {
-                if !event_loop_target.exiting()
-                    && ALL_WINDOWS.with(|windows| {
-                        windows.borrow().iter().any(|(_, w)| {
-                            w.upgrade()
-                                .and_then(|w| {
-                                    w.window().has_active_animations().then(|| {
-                                        w.request_redraw();
-                                        true
-                                    })
-                                })
-                                .unwrap_or_default()
-                        })
+                if !event_loop_target.exiting() {
+                    ALL_WINDOWS.with(|windows| {
+                        for w in windows.borrow().iter().filter_map(|(_, w)| w.upgrade()) {
+                            if w.window().has_active_animations() {
+                                w.request_redraw();
+                            }
+                        }
                     })
-                {
-                    event_loop_target.set_control_flow(ControlFlow::Poll);
```

Comment:
We rely on request_redraw to work correctly so we don't need to poll

Reply:
Agreed. This used to be broken, but I think many good fixes were done in winit to ensure it works nowadays.